cirq/ops/thermal_relaxation_channel_test.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Iterable
+import numpy as np
+import pytest
+
+import cirq
+
+
+def apply_channel(rho: np.ndarray, chan: Iterable[np.ndarray]) -> np.ndarray:
```

Comment:
We should probably use the [apply channel protocol](https://github.com/quantumlib/Cirq/blob/master/cirq/protocols/apply_channel_protocol.py).

Reply:
changed.